deltacat/utils/placement.py

```diff
@@ -229,8 +229,13 @@ def pgs(self):
 
     def get_current_node_resource_key(self) -> str:
         # on ec2: address="172.31.34.51:6379"
-        # on manta: address = "2600:1f10:4674:6815:aadb:2dc8:de61:bc8e:6379"
-        current_node_name = ray.experimental.internal_kv.global_gcs_client.address[:-5]
+        # on AWS Glue for Ray: address = "2600:1f10:4674:6815:aadb:2dc8:de61:bc8e:6379"
+        (
+            current_node_name,
+            _,
+        ) = ray.experimental.internal_kv.global_gcs_client.address.rsplit(
+            ":", 1
+        )  # using rsplit split on the last occurence of delimiter ":"
```

Comment:
These are hacks to get it working, we may soon find another type of URL (say, that doesn't contain port) that would break. How about using `urllib.parse` module?

Reply:
Nevermind, even `urllib.parse` doesn't work above two scenarios.